tools/server_lib/lister.py

```diff
@@ -66,7 +66,7 @@ def list_files(targets=[], ftypes=[], use_shebang=True, modified_only=False,
     # sys.argv as str, so that battle is already lost.  Settle for hoping
     # everything is UTF-8.
     repository_root = subprocess.check_output(['git', 'rev-parse', '--show-toplevel']).strip().decode('utf-8')
-    exclude_abspaths = [os.path.join(repository_root, fpath).rstrip('/') for fpath in exclude]
+    exclude_abspaths = [os.path.normpath(os.path.join(repository_root, fpath).rstrip('/')) for fpath in exclude]
```

Comment:
Yup, definitely! I think the only reason I didn't notice any abnormal behavior in the main repo is that malformatted paths don't produce an error. Will make a PR for it.